h2o-py/tests/testdir_algos/gam/pyunit_PUBDEV_7931_gam_null_predictors.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import division
+from __future__ import print_function
+import sys
+sys.path.insert(1, "../../../")
+import h2o
+from tests import pyunit_utils
+from h2o.estimators.gam import H2OGeneralizedAdditiveEstimator
+
+# In this test, we check and make sure Gam can run without predictor columns as long as gam column is specified
+def test_gam_null_predictors():
+    print("Checking null predictor run for binomial")
+    h2o_data = h2o.import_file(pyunit_utils.locate("smalldata/gam_test/gamBinomial1Col.csv"))
+    myY = "response"
+    buildModelMetricsCheck(h2o_data, 'binomial')
+
+    print("Checking null predictor for gaussian")
+    h2o_data = h2o.import_file(
+    path=pyunit_utils.locate("smalldata/gam_test/gamGaussian1Col.csv"))
+    model_test_data = h2o.import_file(pyunit_utils.locate("smalldata/gam_test/predictGaussianGAM.csv"))
+    buildModelMetricsCheck(h2o_data, 'gaussian')
+
+    print("Checking null predictor for multinomial")
+    h2o_data = h2o.import_file(
+    path=pyunit_utils.locate("smalldata/gam_test/gamMultinomial1Col.csv"))
+    model_test_data = h2o.import_file(pyunit_utils.locate("smalldata/gam_test/predictMultinomialGAM.csv"))
+    buildModelMetricsCheck(h2o_data, 'multinomial')
+    
+    print("gam modelmetrics test completed successfully")    
+    
+def buildModelMetricsCheck(train_data, family):
+    x=[]
```

Comment:
formatting

Reply:
I did a reformat.

Reply:
this should have been prefixed with knit: should be `x = []` (missing spaces)